packages/utils/odsp-utils/src/odspDrives.ts

```diff
@@ -120,7 +117,7 @@ export async function getChildrenByDriveItem(
     do {
         const getChildrenResult = await getAsync(url, authRequestInfo);
         if (getChildrenResult.status !== 200) {
-            throw createErrorFromResponse("Unable to get children", getChildrenResult);
+            throw throwOdspNetworkError("Unable to get children", getChildrenResult);
```

Comment:
'throw throwOdspNetworkError' does not make sense - only one throw should be on the line :)

Reply:
fixed